agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -425,7 +437,7 @@ private static Set<Feature> initStatsbeatFeatureSet(Configuration config) {
     if (config.preview.browserSdkLoader.enabled) {
       featureList.add(Feature.BROWSER_SDK_LOADER);
     }
-    if (!config.preview.sampling.overrides.isEmpty()) {
+    if (!config.sampling.overrides.isEmpty()) {
```

Comment:
Many users have configured the sampling override.

Should the next release warns about deprecation, a following one removes `preview`?

It may also break the behaviors for App service users with the automatic agent upgrade.

Reply:
I don't think this change breaks anything, see 

https://github.com/microsoft/ApplicationInsights-Java/blob/84d364409bbd0264c69fb37d10847bbd06c3d672/agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java#L208-L212